### PR TITLE
Updated README.md & CITATIONS.md & added workflow images

### PR DESCRIPTION
**Pull request by CarsonJM**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phageannotator/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phageannotator _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/images/nf-core-phageannotator_workflow.svg
Comment (maxulysse) [inline docs/images/nf-core-phageannotator_workflow.svg:1]: Lots of empty space at the top and bottom of the picture.
I'd recommend resizing the page to the drawing only

**Comment by CarsonJM** [comment]
Is this new workflow schematic more what you were looking for?

File: docs/images/nf-core-phageannotator_workflow.png
Comment (maxulysse) [inline docs/images/nf-core-phageannotator_workflow.png:1]: small transparent outline on up and right side

**Comment by maxulysse** [comment]
> Is this new workflow schematic more what you were looking for?

Not really, the global size hasn't changed, check in the file changes tab here on github, you might see what I'm talking about there.

**Comment by CarsonJM** [comment]
Sorry, but I'm still missing something! I can see the transparent outlines, but am unsure what you mean about resizing the page. Are you recommending that I crop any borders in the image or just make the image smaller? Thanks for bearing with me!

**Comment by maxulysse** [comment]
I made screenshots from the `Files Changed` tab here on GitHub.

So first the PNG, you have a small transparent border on the right and upper side:
png

Second the SVG, the document size compared to the image size doesn't really compare, lots of empty space on top and on the bottom:
svg

**Comment by CarsonJM** [comment]
Thanks for the screenshots. That is very clear. For some reason the large SVG borders are not rendering on my end!
image

**Review by maxulysse — APPROVED** [review]
LGTM

**Comment by CarsonJM** [comment]
Another basic question: Do I set up and run checks on my end, or is that something done on the nf-core side? Right now it seems as though I can't start workflows

**Pull request merged**